package.json

```diff
@@ -196,7 +196,6 @@
     "@types/promise-retry": "^1.1.3",
     "@types/promise-timeout": "^1.3.0",
     "@types/puppeteer-core": "^5.4.0",
-    "@types/react": "^18.2.73",
```

Comment:
IIRC this is used here `src/app/views/templates/MrfWorkflowEmail.tsx`

Reply:
Why got tsx in serverrrrrrrr

Reply:
Email generation. You have ideas to avoid needing the `react` types here?

Reply:
new monorepo package with react-email, build via `email export`, then copy the built into the server repo (via package's script) would be my suggestion. I will see if I can do that first then.

Reply:
for now i'll sync the types to `/frontend`'s folder so lint works properly

Reply:
doesn't work, i'll do the monorepo package thingy first then....

Reply:
i think i'm going down the route that @justynoh took lmao. need dynamic data, and a whole pipeline. I'll deprioritise it. 

(doable though, can probably just render as static, and use express's templating engine for the data instead of react-email)

Reply:
ya it's possible to just use the templating engine but every time we make a change to the email format, will need to regenerate the raw template. This was at the time i was trying to play around with different formats and send to GSIB outlook to see how they would be rendered, and I didn't really want to have to keep re-exporting every time i make one small change.

Reply:
fixed it by adding the `react-email` package to root, so the types work. (and correctly, since the server is also the one rendering the email, should really have the package)